cis194/week1/andrew/Homework.hs

```diff
@@ -0,0 +1,43 @@
+module Homework
+    ( toDigits,
+      toDigitsRev
```

Comment:
check this best practise:
https://github.com/fp-works/2019-winter-Haskell-school/pull/7#discussion_r312690736

Reply:
sure will do.